tools/mhm-tools/mhm_bankfull_discharge/bankfull_discharge.xml

```diff
@@ -0,0 +1,70 @@
+<tool id="bankfull_discharge" name="Calculate Bankfull Discharge and Wetted Perimeter" version="0.1.0+galaxy0" python_template_version="3.5" profile="21.05">
+    <description>
+        Calculate the river discharge at bankfull conditions and the bankfull width.
+    </description>
+    <requirements>
+        <requirement type="package" version="3.8">python</requirement>
```

Comment:
`mhm` should be the requirement .. then remove all the copied python code? Or am I missing something?